druid-shell/src/platform/x11/application.rs

```diff
@@ -126,13 +138,27 @@ impl Application {
             }
         };
 
+        let handle = x11rb::cursor::Handle::new(connection.as_ref(), screen_num, &rdb)?.reply()?;
+        let load_cursor = |name| handle.load_cursor(connection.as_ref(), name);
+
+        let cursors = Cursors {
+            default: load_cursor("default")?,
+            text: load_cursor("text")?,
+            pointer: load_cursor("pointer")?,
+            crosshair: load_cursor("crosshair")?,
+            not_allowed: load_cursor("not-allowed")?,
+            row_resize: load_cursor("row-resize")?,
+            col_resize: load_cursor("col-resize")?,
```

Comment:
Is there a chance that these loads could fail? My understanding is that because these belong to the default cursor font, the only way they could fail is through some catastrophic failure (e.g. losing the connection). If this is the case, it's probably worth noting it in a comment. If I'm mistaken and there are other failure modes, I think we just want to fail to load that single cursor instead of aborting the whole function.

Reply:
after taking a look at x11rb cursor loading code, even though these are in default fonts, but are named differently in the default font.

It makes sense to make all these `Option<xproto::Cursor>` and then not setting the cursor if it is not loaded and `warn`ing.